core/mondoo-okta-security.mql.yaml

```diff
@@ -1,379 +1,1985 @@
+
+            ```
+            resource "okta_idp_oidc" "example" {
+              name                  = "example"
+              authorization_url     = "https://idp.example.com/authorize"
+              authorization_binding = "HTTP-REDIRECT"
+              token_url             = "https://idp.example.com/token"
+              token_binding         = "HTTP-POST"
+              user_info_url         = "https://idp.example.com/userinfo"
+              user_info_binding     = "HTTP-REDIRECT"
+              jwks_url              = "https://idp.example.com/keys"
+              jwks_binding          = "HTTP-REDIRECT"
+              scopes                = ["openid"]
+              client_id             = "efg456"
+              client_secret         = "efg456"
+              issuer_url            = "https://id.example.com"
+              username_template     = "idpuser.email"
+            }
+            ```
+
+
+  - uid: mondoo-okta-security-enable-suspicious-activity-reporting
+    title: Enable Suspicious Activity Reporting 
+    impact: 80
+    docs: 
+      desc: |
+        When a user reports suspicious activity, admins can enable specific actions and System Log events to obtain further details about the activity.
+
+        ### Okta Recommends 
+
+        Enable Suspicious Activity Reporting for end-user reporting.
+
+        ### Security impact 
 
-        Run this query:
+        High
 
-          ```bash
-          cnspec run okta -c "okta.policies.signOn.where(_.rules.all(status == "ACTIVE")).all(_.rules.all(actions["signon"]["factorPromptMode"] == "ALWAYS"))" --organization <YOUR-ORG> --token <OKTA_TOKEN>
-          ```
+        ### User impact 
 
-        __Use the cnspec shell__
+        Low 
+    variants: 
+      - uid: mondoo-okta-security-enable-suspicious-activity-reporting-runtime
+      - uid: mondoo-okta-security-enable-suspicious-activity-reporting-terraform-hcl
+      - uid: mondoo-okta-security-enable-suspicious-activity-reporting-terraform-plan
+  - uid: mondoo-okta-security-enable-suspicious-activity-reporting-runtime 
+    title: Enable Suspicious Activity Reporting 
+    impact: 80
+    filters: asset.platform == "okta-org"
+    mql: okta.organization.securityNotificationEmails['reportSuspiciousActivityEnabled'] == true
+    docs:
+      remediation: 
+        - id: console 
+          desc: |
+            ### Enable or disable Security Notification emails
+            If you disable this feature, all valid links expire immediately.
+
+            If you disable the Report suspicious activity via email option, the Report Suspicious Activity button is removed from the email templates that use it.
+
+            When you enable the Report suspicious activity via email option, events reported when users click the Report Suspicious Activity button appear on the Admin Console. Click Review Security Event to view the event details in the System Log. The event name is:
```

Comment:
```suggestion
            When you enable the Report suspicious activity via email option, events reported when users select the Report Suspicious Activity button appear on the Admin Console. Click Review Security Event to view the event details in the System Log. The event name is:
```

Reply:
@tas50 these are not our steps. they are from Okta's docs. They are copy/pasted.